src/main/java/me/totalfreedom/totalfreedommod/command/Command_coins.java

```diff
@@ -9,8 +9,11 @@
 import org.bukkit.command.CommandSender;
 import org.bukkit.entity.Player;
 
+import java.util.ArrayList;
+import java.util.List;
+
 @CommandPermissions(level = Rank.OP, source = SourceType.BOTH)
-@CommandParameters(description = "Shows the amount of coins you have or another player has", usage = "/<command> [playername]")
+@CommandParameters(description = "Shows the amount of coins you or another player has. ", usage = "/<command> [player] | pay <player> <amount>")
```

Comment:
Could you update the description to be more accurate to reflect the fact it won't just show coin values.

Reply:
good catch, though i'm not exactly sure how i'd word it